lang/en.json

```diff
@@ -168,6 +168,8 @@
   "POWER_LEVEL": "Power Level",
   "SUCCEED": "Succeed",
   "FAILED": "Failed",
+  "OTHER": "OTHER",
+  "CARRYING_CAPACITY": "Carrying capacity",
```

Comment:
The existing translations mostly seems to follow this idea of categorisation with a common  prefix to related strings. I would probably try to follow that and put these strings as `ITEM.OTHER` and `ITEM.CARRYING_CAPACITY`. But don't know what you think? Since it isn't consistent there's no clear right or wrong, I guess.

Reply:
I think that those were referring to the ITEM tab and what is found on there. Originally I wanted to put it into GEAR but I'm not sure how it would fit into the current categorization system so I added them to the general set.